script.js

```diff
@@ -41,7 +69,7 @@ var getProgramMove = function () {
 
   switch (move) {
     case 0:
-      return "scissors";
+      return "scissors";gi
```

Comment:
Pleasantly surprised to see a switch statement! There might be a typo on line 72 though.